lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1296,7 +1204,7 @@ public function isIdentifier(string $fieldName): bool
      *
      * @return bool TRUE if the field is unique, FALSE otherwise.
      */
-    public function isUniqueField($fieldName)
+    public function isUniqueField(string $fieldName): bool
```

Comment:
`@param` can be removed. `@return` too.

Reply:
I hadn't reevaluated the comments but indeed, it's pretty tautological :sweat_smile: